payments/trustly/seamless-view.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Swedbank Pay Trustly Payments Seamless view
+sidebar:
+  navigation:
+  - title: Trustly Payments
+    items:
+    - url: /payments/trustly
+      title: Introduction
+    - url: /payments/trustly/redirect
+      title: Redirect
+    - url: /payments/trustly/seamless-view
+      title: Seamless View
+    - url: /payments/trustly/after-payment
+      title: After Payment
+    - url: /payments/trustly/other-features
+      title: Other Features
+---
+
+{% include jumbotron.html body="The Seamless View scenario represents the
+                                opportunity to implement Trustly directly in
+                                your webshop." %}
+
+## Introduction
+
+Seamless View provides an integration of the payment process directly on your
+website. This solution offers a smooth shopping experience with Swedbank Pay
+Payments seamlessly integrated in an `iframe` on your website. The payer
+does not need to leave your webpage, since we are handling the payment in the
+`iframe` on your website.
+
+![screenshot of the Trustly payment window][trustly-payment-embedded-view]{:height="425px" width="700px"}
+
+{% include alert-callback-url.md api_resource="trustly" %}
+
+## Step 1: Create the payment
+
+A Trustly payment is a straightforward way to perform a direct-bank payment. It is followed up by posting a cancellation or reversal
```

Comment:
```suggestion
A Trustly payment is a straightforward way to perform a direct-bank payment. It can be followed up by posting a cancellation or reversal
```